libpay-braintree/src/test/scala/com/wix/pay/braintree/it/BraintreeGatewayWithOAuthIT.scala

```diff
@@ -0,0 +1,46 @@
+package com.wix.pay.braintree.it
+
+import com.braintreegateway.Environment
+import com.wix.pay.PaymentGateway
+import com.wix.pay.braintree._
+import com.wix.pay.creditcard.{CreditCard, YearMonth}
+import com.wix.pay.model.CurrencyAmount
+import org.specs2.mutable.SpecWithJUnit
+
+
+class BraintreeGatewayWithOAuthIT extends SpecWithJUnit {
```

Comment:
I do not think this would be a good idea.
Remember that this is an open source project; I do not think we want to put Wix credentials to Braintree Sandbox.
And regardless, and as Danny mentioned, as an open source, this should be the state of the art of Wix engineering. Using the sandbox is not a good example from TDD, which we preach on at Wix.